code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -103,6 +106,8 @@
 	var/health_cost = 0
 	/// Have we already been positioned into our starting location?
 	var/positioned = FALSE
+	/// If FALSE, does not delete after the spell ran out of charges. Used on blood rites.
+	var/deletes_on_empty = TRUE
```

Comment:
I don't think you've applied this new var to blood rites.

Reply:
oh yeah i tested cult stun but forgor to test blood rites